pkg/controller/controller_binding.go

```diff
@@ -398,14 +439,25 @@ func (c *controller) reconcileServiceInstanceCredential(binding *v1alpha1.Servic
 				"Bind call failed. "+s)
 
 			if !time.Now().Before(toUpdate.Status.OperationStartTime.Time.Add(c.reconciliationRetryDuration)) {
-				s := fmt.Sprintf(`Stopping reconciliation retries on ServiceInstanceCredential "%v/%v" because too much time has elapsed`, binding.Namespace, binding.Name)
-				glog.Info(s)
-				c.recorder.Event(binding, api.EventTypeWarning, errorReconciliationRetryTimeoutReason, s)
-				c.setServiceInstanceCredentialCondition(toUpdate,
-					v1alpha1.ServiceInstanceCredentialConditionFailed,
-					v1alpha1.ConditionTrue,
-					errorReconciliationRetryTimeoutReason,
-					s)
+				if toUpdate.Status.OrphanMitigationInProgress {
```

Comment:
This check is not needed as `OrphanMitigationInProgress` will never be `true` here.

Reply:
Somehow this comment is now applying to the wrong area of code, but the correction was made.